docs/docs/features/underglow.md

```diff
@@ -167,10 +192,11 @@ Once you have your `led_strip` properly defined you need to add it to the root d
 };
 ```
 
-Finally you need to enable the `CONFIG_ZMK_RGB_UNDERGLOW` and `CONFIG_*_STRIP` configuration values in the `.conf` file of your board (or set a default in the `Kconfig.defconfig`):
+Lastly, [enable the feature](#enabling-rgb-underglow). This can also be done in `Kconfig.defconfig`:
 
-```ini
-CONFIG_ZMK_RGB_UNDERGLOW=y
-# Use the STRIP config specific to the LEDs you're using
-CONFIG_WS2812_STRIP=y
+```kconfig
+config ZMK_RGB_UNDERGLOW
+  default y
+  # If not using WS2812 LEDs, imply the appropriate *_STRIP.
+  imply WS2812_STRIP
 ```
```

Comment:
I'm iffy about this, as it blurs the lines between keyboard users and keyboard designers a bit too much for my taste. I think we want designers to make the feature available, but not enable it, and the kconfig.defconfig should be written accordingly.

Reply:
Same 4-space indent comment here.